README.md

```diff
@@ -1,2 +1,32 @@
-# school_library
-Using OOP principles to build a school library app that contains Students, Teachers and other information
+# school_library
+Using OOP principles to build a school library app that contains Students, Teachers and other information
+
+## Built with
+- Ruby
+
+## Getting Started
+Clone the repository using `git clone` then open terminal and enter `cd school_library`

```

Comment:
[ ] OPTIONAL  to make your project more professional, kindly add the command to run the APP: `ruby main to the README file.rb`